scripts/pre_commit_linter.py

```diff
@@ -67,8 +67,27 @@
 import threading
 import time
 
-import docstrings_checker  # pylint: disable=relative-import
+from . import docstrings_checker
+from . import python_utils
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_FUTURE_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'future-0.17.1')
+
+sys.path.insert(0, _FUTURE_PATH)
+
+# pylint: disable=wrong-import-position
+# pylint: disable=wrong-import-order
+from builtins import object  # isort:skip
```

Comment:
Hm. Is "object" a module? If not, how about "import builtins" and then use builtins.object?

We should always import at the module level.

Reply:
We can use `builtins.object`, `builtins.range` etc but that would mean we have to write builtins.range everywhere in the codebase instead of the regular range. This can be really inconvenient for developers and can create confusion. Also after the GAE dependency issues are resolved removing Python 2 backwards compatibility would be tedious(if that's something we wanna do)

Reply:
I am fine with using builtins.object etc. I don't see why that would affect the rest of the codebase rather than just this file, though. Could you explain?

Also, could you explain why we need to do this in the first place? What is wrong with the regular `object`?

Finally, why would removing Python 2 be tedious? https://docs.python.org/3/library/builtins.html suggests that this is how builtins are invoked in Python 3.

Reply:
No I just meant developers would have to write builtins.range everywhere instead of just range and that could be tedious.

I just know that some inbuilt methods(like `next -> __next__`) are renamed in Python 3 and the `builtins.object` takes care of that behaviour. You can look at [this SO answer](https://stackoverflow.com/a/36517807) for more details.

Finally I just meant that if we wanna remove the whole builtins module from the codebase later on we will again have to modify builtins.range -> range and so on.

Still, I have made the changes!

Reply:
Hi @lilithxxx, you haven't answered my previous comment fully. Could you please make sure you have addressed all of the questions? (Make sure to do this in general, otherwise it delays the review cycle.)

Reply:
Hi @seanlip yes `object` is a module and I made the changes to import at module level. Is there any more comment that I forgot to address? Thanks

Reply:
Ah, sorry, this one was my mistake. I thought that this part of the comment was unaddressed:

> Also, could you explain why we need to do this in the first place? What is wrong with the regular object?

but I guess your second paragraph actually addresses that. So, this should be resolved, thanks!